### DISCUSSION TITLE
Difficulty with determining beta with transverse flutes

### DISCUSSION THREAD

**Discussion by edwardkort** [edwardkort/WWIDesigner#95] [Flute Study] [answered]
From Bamboo Flute:
Can you please have some exercise through transverse flute. Fipple  / Beta factor is difficult to find out or may be this program have limitations and not be able to find correct beta factor. Thank you.

**Comment by bhp1** [comment, marked as answer]
I suggest you start with beta set to 0.3.  To know how good that guess is, you have to compare the tuning the model predicts to the tuning of a real flute.  Create an instrument file that gives the geometry of the real flute, measure the frequencies you get from the real flute when you play it, and follow the process described in https://github.com/edwardkort/WWIDesigner/wiki/Working-with-the-Flute-Study-Model#calibrating-the-flute-model.
